### PR TITLE
chore(report): REST APIs week 1  & 2 report

### PR DESCRIPTION
**Pull request by valens200**
## Project

REST API improvements.

### Changes
- Changed the folder structure by adding the folder holding my updates for REST API improvements.
- Added updates for weeks 1, 2, and 3 for REST API improvements.

### CODE REVIEW

File: docs/2024/text-phrases/index.md [hunk 1]
```diff
@@ -3,29 +3,51 @@ sidebar_position: 1
...
 ## Author
 
-[Valens Niyonsenga](https://github.com/valens200)
+[Valens NIYONSENGA](https://github.com/valens200)
 
 ## Contact info
 
-- [Email](mailto:email.here)
-- [LinkedIn](https://linkedin.com/in/my-user)
+- [Email](valensniyonsenga2003@gmail.com)
```
Comment (GMishx) [inline docs/2024/text-phrases/index.md:19]: Please change to email link as it causes build failure.

```suggestion
- [Email](mailto:valensniyonsenga2003@gmail.com)
```

**Review by GMishx — CHANGES_REQUESTED** [review]
Minor change needed.

@valens200 please rebase the branch with main and move other files created by @dvjsharma as well.

File: docs/2024/text-phrases/index.md (hunk 1 above)
Comment (valens200) [reply docs/2024/text-phrases/index.md:19]: Sure @GMishx I will take a look into this.

**Comment by valens200** [comment]
> Minor change needed.
> 
> @valens200 please rebase the branch with main and move other files created by @dvjsharma as well.
Thank you @GMishx, I have rebased the branch and moved other files from Sharma.

**Pull request merged**